### PR TITLE
BOM-470

### PR DESCRIPTION
**Pull request by awais786**
Updated the xblock-sdk hash.
Making this repo python3 compatible.

### CODE REVIEW

**Comment by awais786** [comment]
PR is failing on python3.5. In this screen shot I have attached python2 and python3 windows.
Python3 is not loading the objects.

Screen Shot 2019-10-02 at 4 58 30 PM

File: tests/utils.py [hunk 1]
```diff
@@ -18,7 +19,11 @@ def make_request(data, method='POST'):
     """ Make a webob JSON Request """
     request = Request.blank('/')
     request.method = 'POST'
-    request.body = json.dumps(data).encode('utf-8') if data is not None else ""
+    data = json.dumps(data).encode('utf-8') if data is not None else ""
```
Comment (feanil) [inline tests/utils.py]: Why not just make the else clause like `else b""`  which will provide bytes version of json string or an empty bytestring?

**Comment by ihtram** [comment]
> o you have any tips or can you hel

**Comment by awais786** [comment]
@feanil I cloned the PR with py3.5 environment. 
```./install_test_deps.py```
You have to do some travis.yml steps also.
```
    pip uninstall -y selenium
    pip install selenium==2.53.6
```
Run this command to make sure units are tests are running fine`python run_tests.py tests/unit/`
For bokchoy tests run this command `python run_tests.py tests/integration/` It will execute all tests and takes lots of time so you need to run the test_sizing.py tests only for debugging.

**Comment by awais786** [comment]
@feanil @ihtram I just fixed some thing lets see how it behaves.

File: tests/integration/test_sizing.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def _svg_to_data_uri(path):
     data_path = os.path.dirname(__file__) + "/data/"
     with open(data_path + path, "rb") as svg_fh:
         encoded = base64.b64encode(svg_fh.read())
-    return "data:image/svg+xml;base64,{}".format(encoded)
+    return "data:image/svg+xml;base64,{}".format(encoded.decode('utf-8'))
```
Comment (awais786) [inline tests/integration/test_sizing.py]: @feanil I made this change. `b` was appearing with image data.
```
data:image/svg+xml;base64,b&#39;PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxOS4xL
```

**Comment by awais786** [comment]
@feanil plz review again.

File: openedx.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: xbdndv2
+owner: unknown
```
Comment (feanil) [inline openedx.yaml]: You can just not put in the `nick` and `owner` settings.

**Pull request merged**